charts/tempo-distributed/templates/compactor/deployment-compactor.yaml

```diff
@@ -70,9 +70,14 @@ spec:
               name: http-metrics
             - containerPort: {{ include "tempo.memberlistBindPort" . }}
               name: http-memberlist
-          {{- with .Values.compactor.extraEnv }}
```

Comment:
Is there any reason you changed `.Values.compactor.extraEnv` to `.Values.compactor.env` in the compactor but for distributor you left it the same?
This is just an example, other components are a bit inconsistent as well.
I think we should keep `.Values.compactor.extraEnv` as is not a breaking change.

I realize is just a copy paaste issue because you used `{{- with .Values.compactor.extraEnv }}` inside the if.

Reply:
@andreigorgan Thanks for pointing this out, I'll patch it by opening a PR